x-pack/plugins/siem/scripts/endpoint/resolver_generator.ts

```diff
@@ -70,7 +70,7 @@ async function main() {
     metadataIndex: {
       alias: 'mi',
       describe: 'index to store host metadata in',
-      default: 'metrics-endpoint-default-1',
+      default: 'metrics-endpoint.metadata-default-1',
```

Comment:
ahhh. I just hit this problem (meaning: generator stopped working - or better said: was working, but placing docs in incorrect index).

Would we be able to use the input args to do a query to Ingest to get the index pattern from there? It would make changes to them transparent to the generator going forward.

Reply:
Yes, we plan to remove our mapping et al' later. For now we are just wiring thing up.

Reply:
Yeah that's a great idea. It's something that we've wanted to add, just haven't had a chance yet. 👍